### ISSUE TITLE
WidgetExt::connect_drag_data_get needs mutable arguments

### ISSUE THREAD

**Issue by Susurrus** [gtk-rs/gtk#474]
The `&SelectionData` argument needs to be passed mutably because it's also an output. The following code should be valid:
```
button.connect_drag_data_get(|_, _, s, _, _| {
    let data = "i'm data!";
    s.set_text(data, data.len());
});
```

**Comment by EPashkin** [comment]
Then it need be done manually, as IMHO gir currently don't support IN-OUT parameters at all

**Comment by Susurrus** [comment]
This was resolved in #472.
